features/metadata.feature

```diff
@@ -35,7 +35,6 @@ Feature: Metadata
     Background:
         Given a metadata starts with an empty object
 
-    @ignore
     Scenario Outline: Adding some data to metadata
```

Comment:
`Adding some data to metadata` and `Store a record of the metadata when the event is done` look very similar. They are all checking if the meta is set in the subsequent builds. Can we combine them somehow?